framework/src/Volo.Abp.MongoDB/Volo/Abp/Domain/Repositories/MongoDB/IMongoDbRepositoryFilterer.cs

```diff
@@ -14,5 +14,26 @@ public interface IMongoDbRepositoryFilterer<TEntity, TKey> : IMongoDbRepositoryF
         FilterDefinition<TEntity> CreateEntityFilter(TKey id, bool applyFilters = false);
 
         FilterDefinition<TEntity> CreateEntityFilter(TEntity entity, bool withConcurrencyStamp = false, string concurrencyStamp = null);
+
+        /// <summary>
+        /// Creates 'In' filter for mongoDb.
```

Comment:
Interfaces should not give information on the implementation details. If you want to give it, you can write "by default", like `Creates 'In' filter by the default implementation` (no need to write MongoDB, it is already MongoDB).

Reply:
Can you check Commit f3722985